docs/references/general/rawacf.md

```diff
@@ -1,3 +1,110 @@
+| *lagfr*                 | *us*     | short     | Lag to first range                                                                      |
+| *smsep*                 | *us*     | short     | Sample seperation                                                                       |
+| *ercod*                 | *None*   | short     | Error Code                                                                              |
+| *stat.agc*              | *None*   | short     | AGC status word                                                                         |
+| *stat.lopwr*            | *None*   | short     | LOPWR status word                                                                       |
+| *noise.search*          | *None*   | float     | Calculated noise from clear frequency search                                            |
+| *noise.mean*            | *None*   | float     | Average noise from clear frequency band                                                 |
+| *channel*               | *None*   | short     | Channel number, used to denote different Tx/Rx channels on Stereo radars, and to denote changes in radar operating parameters between scans, e.g. alternating between 2 frequencies scan-to-scan      |
+| *bmnum*                 | *None*   | short     | Beam number, zero based indexing                                                        |
+| *bmazm*                 | *degrees*   | float     | Beam azimuth                                                                            |
+| *scan*                  | *None*   | short     | Scan flag, switches between 0 to +/-1. For some radars, the sign indicates scan direction. |
+| *offset*                | *us*     | short     | Offset between channels for a stereo radar                                              |
+| *rxrise*                | *us*     | short     | Receiver rise time                                                                      |
+| *intt.sc*               | *s*      | short     | Number of seconds of integration time                                                   |
+| *intt.us*               | *us*     | int       | Number of microseconds of integration time                                              |
+| *txpl*                  | *us*     | short     | Transmitted pulse length                                                                |
+| *mpinc*                 | *us*     | short     | Multi-pulse increment                                                                   |
+| *mppul*                 | *None*   | short     | Number of pulses in the sequence                                                        |
+| *mplgs*                 | *None*   | short     | Number of lags estimated from the sequence                                              |
+| *mplgexs*               | *None*   | short     | Number of lags used for tauscan. This field originates from `fittex` algorithm and is not used in RAWACF file format. | 
```

Comment:
```suggestion
| *mplgexs*               | *None*   | short     | Number of lags used for tauscan. This field originates from `fitex` algorithm and is not used in RAWACF file format. | 
```

Is it true that the `mplgexs` field is never used in `rawacf` files?  And by that I mean, wouldn't it be possible to produce a `rawacf` file from an `iqdat` file using the full tauscan sequence?

Reply:
When I ran my script for DSWG to find all fields in RAWACF and Optional ones I got these fields. 
For Example: 
20200101.0001.00.cve.rawacf.bz2 had: `xcfd`, `ifmode`, `mplgexs`, `acfd`

So they get documented. I think the question "if all these fields are valid in RAWACF file?" might be more useful to be addressed to DSWG.

Reply:
@mts299 sorry but that's not quite what I was getting at.  Presumably, one could produce a `rawacf` file from an `iqdat` file which contains both the forward and reverse tauscan-style pulse sequence using the `acfex` library rather than the usual `acf` library.  In that case, the `mplgexs` field of the `rawacf` file could be important for the end user.  I am also not sure if it is entirely accurate to say that the field originates from the `FITEX` algorithm.  Maybe the second sentence of the description could be completely removed?

Reply:
Yea sure use the suggestion notation to suggest it and see what people think.

Reply:
Looks like there's still something here for a suggestion on how to reword it for not including the `fitex`.  Maybe I can take a crack at it if needed.